gpt_engineer/steps.py

```diff
@@ -38,21 +38,21 @@ def get_prompt(dbs: DBs) -> str:
 # All steps below have the signature Step
 
 
-def simple_gen(ai: AI, dbs: DBs) -> List[dict]:
+def simple_gen(ai: AI, dbs: DBs, step_name:str) -> List[dict]:
```

Comment:
```suggestion
import inspect

def curr_fn() -> str:
    return inspect.currentframe().f_code.co_name

def simple_gen(ai: AI, dbs: DBs) -> List[dict]:
```

Reply:
We need to use
```
def curr_fn() -> str:
    return inspect.stack()[1].function
```
to get the current function's name. Your suggestion would always return "curr_fn". But done ✅